lib/outbound/index.js

```diff
@@ -42,25 +42,27 @@ Outbound.prototype.loaded = function(){
 };
 
 Outbound.prototype.identify = function(identify){
-  var firstLevelAttributes = ['user_id', 'first_name', 'last_name', 'email', 'phone_number', 'apns', 'gcm'];
-  var allAttributes = identify.traits();
-  var userAttributes = {
-    attributes: {}
+  var userAttributes = function(identify){
+    var body = {
+      user_id: identify.traits().id,
+      attributes: identify.traits(),
+      email: identify.traits().email,
+      phone_number: identify.traits().phone,
+      first_name: identify.traits().firstName,
+      last_name: identify.traits().lastName
+    };
+
+    var attrs = body.attributes;
+
+    delete attrs.firstName;
+    delete attrs.lastName;
+    delete attrs.id;
+    delete attrs.phone;
+    delete attrs.email;
+    return body;
```

Comment:
Cache the results of calling `traits()` here; otherwise you're making a new copy of the traits object on every invocation of `traits()`:

```js
var traits = identify.traits();

// Now use any values off `traits` you need
```

Also, make a copy of attrs rather than using `delete`:

```javascript
var omit = require('omit');

// ...
return {
  user_id: traits.id,
  attributes: omit(['id', 'phone', 'firstName', 'lastName'], traits),
  email: traits.email,
  phone_number: traits.phone,
  first_name: traits.firstName,
  last_name: traits.lastName
};
```